src/service/Service.cmake

```diff
@@ -105,6 +105,8 @@ target_link_libraries(${NEBULA_SERVICE}
     PUBLIC ${MSGPACK_LIBRARY}
     PUBLIC ${LEVELDB_LIBRARY})
 
+add_dependencies(${NEBULA_SERVICE} compile_fbs)
```

Comment:
this is bug fix - master not able to build

Reply:
yeah, otherwise need to run below command manually
```./build/flatbuffers/src/flatbuffers-build/flatc -b -o "./src/service/gen/nebula/node" --cpp --grpc "./src/service/fbs/node.fbs"```